WcaOnRails/.rubocop.yml

```diff
@@ -21,6 +21,9 @@ Lint/UnusedBlockArgument:
 Lint/UnusedMethodArgument:
   Enabled: false
 
+Style/AccessorMethodName:
+  Enabled: false
```

Comment:
Because I don't want to deal with this stuff:

```
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> be rubocop
Inspecting 417 files
...................................................................................................................................................................................................................C.............................................................................................................................................................................................................

Offenses:

app/models/competition.rb:827:7: C: Do not prefix writer method names with set_.
  def set_wcif!(wcif)
      ^^^^^^^^^
app/models/competition.rb:832:7: C: Do not prefix writer method names with set_.
  def set_events!(wcif_events)
      ^^^^^^^^^^^

417 files inspected, 2 offenses detected
```

The alternative would be to make a `def wcif=`, but I don't think that conveys the fact that we're writing to the database and might blow up in the same way that a `!` does.